src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbbreviationAsWordInNameCheck.java

```diff
@@ -154,6 +158,29 @@
  *   String firstXML; // violation, XML abbreviation is not allowed
  * }
  * </pre>
+ * <p>
+ * To configure to check variables, enforcing no abbreviations
+ * except for variables that are both static and final (e.g. constants).
```

Comment:
`(e.g. constants)` I think this is redundant, there are no other options :)

Reply:
From https://github.com/checkstyle/checkstyle/issues/6582#issuecomment-575938195:
> https://checkstyle.org/google_style.html#a5.2.4
> > Every constant is a static final field, but not all static final fields are constants - impossible to check such rule.
> 
> If we add it, there will be false positives that go against google's rules. The guide shows some specific examples like
> > static final Set mutableCollection = new HashSet();

Reply:
it is better to remove `(e.g. constants)` , this term a bit broad, some people by such term mean non-changed value and immutable inside(if it is reference to collection or object), some people just non-changed value/reference.

Reply:
Fixed